src/main/webapp/app/exam/manage/exam-management.service.ts

```diff
@@ -1,8 +1,111 @@
 import { Injectable } from '@angular/core';
-
+import { Router } from '@angular/router';
+import { HttpClient, HttpResponse } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import * as moment from 'moment';
 import { SERVER_API_URL } from 'app/app.constants';
+import { Exam } from 'app/entities/exam.model';
+import { createRequestOption } from 'app/shared/util/request-util';
+
+type EntityResponseType = HttpResponse<Exam>;
+type EntityArrayResponseType = HttpResponse<Exam[]>;
 
 @Injectable({ providedIn: 'root' })
 export class ExamManagementService {
-    public resourceUrl = SERVER_API_URL + 'api/exams';
+    public resourceUrlCourses = SERVER_API_URL + 'api/courses';
```

Comment:
we should always use 'api/courses' as base URL. This simplifies authorization checks on the server! Only instructors and admins should be able to call requests related to exams